network_path/data/conf.yaml.example

```diff
@@ -0,0 +1,56 @@
+init_config:
```

Comment:
Hum, that's strange, I was expecting [conf.yaml.example](https://github.com/DataDog/datadog-agent/pull/27920/files#diff-1d825d645a6aa94667046655aaf25ea89631a33c8ecda070aca41f8b05ef8ae8R1) in datadog-agent to be present in datadog/agent:7.57.0-rc.1 

For SNMP, we have it in integrations-core for historical reasons since SNMP started being a python integration.